examples/transpiler/babel.bzl

```diff
@@ -35,7 +43,16 @@ def babel(name, srcs, js_outs, map_outs, **kwargs):
                 "//examples:node_modules/@babel/preset-typescript",
             ],
             chdir = "examples",
-            outs = outs,
+            outs = [js_out, map_out],
             args = args,
             **kwargs
         )
+
+        outs.append(js_out)
+        outs.append(map_out)
+
+    # The target producing the js files which ts_project() will reference
+    js_library(
```

Comment:
This is required because the main `js_library` within the `ts_project` macro now includes the compiled js/maps via rule name (this rule), where previously it included them all as file references.

Previously this `name` didn't have to be respected at all actually since everything was referenced as output files instead of rule targets.